autofill/autofill-impl/src/main/java/com/duckduckgo/autofill/impl/ui/credential/saving/AutofillSavingCredentialsViewModel.kt

```diff
@@ -37,7 +37,8 @@ class AutofillSavingCredentialsViewModel @Inject constructor(
     lateinit var autofillStore: AutofillStore
 
     fun showOnboarding(): Boolean {
-        return autofillStore.showOnboardingWhenOfferingToSaveLogin
+        // hardcoding this to always show onboarding-style dialog for now; instead of querying autofillStore.showOnboardingWhenOfferingToSaveLogin
```

Comment:
This is temporary; will check back in later and remove the unused _non-onboarding_ logic if it seems unlikely to be used again. Reminder set ⏰ [Autofill: Remove redundant non-onboarding save login dialog](https://app.asana.com/0/1204099438373202/1204099438373202/f)